compiler/src/iree/compiler/Dialect/Stream/IR/StreamOps.cpp

```diff
@@ -76,6 +76,36 @@ verifyDispatchWorkload(Operation *op, IREE::Stream::ExecutableExportOp exportOp,
   return success();
 }
 
+// Verifies the tied operand types are as the same as the result types.
+static LogicalResult verifyTiedOperandEncodings(Operation *op,
+                                                ArrayAttr operandEncodingsAttr,
+                                                ArrayAttr resultEncodingsAttr) {
+  auto tiedOp = dyn_cast<IREE::Util::TiedOpInterface>(op);
+  if (!tiedOp) {
+    return op->emitOpError()
+           << "the op does not implement IREE::Util::TiedOpInterface";
+  }
+
+  ArrayRef<Attribute> operandEncodings = operandEncodingsAttr.getValue();
+  unsigned tiedOperandBase = tiedOp.getTiedOperandsIndexAndLength().first;
+  for (auto [idx, resEncoding] :
```

Comment:
style: as a general rule err on the side of not abbreviating and keeping things consistent (`op.getResultEncodings()` -> `resultEncodingsAttr` -> `resultEncoding`/`resultEncodingAttr`, not `resEncoding`) - over time as things are renamed if different places use arbitrarily abbreviated names find/replace don't work and things start to drift - keeping the names (either root prefix or suffix) consistent makes it much easier to maintain.
```suggestion
  for (auto [idx, resultEncoding] :
```

Reply:
Sorry that I forgot it when I wrote the code. I do remember that you mentioned the rule in one of the reviews. Done!